README.md

```diff
@@ -19,215 +19,52 @@
+| rpc_addr | 127.0.0.1:3030 | `OPTIONAL` local address to listen for rpc reqs |
+| fee_token_addr | 0x049d36...e004dc7 | `OPTIONAL` fee token to check for `getBalance` |
 
-#### Build
+Direct the beerus cli to your config file as follows:
 
 ```bash
-cargo build --all --release
-```
+cargo build
+./target/debug/beerus -c examples/conf/beerus.json
 
-Build `beerus-core` for WASM:
+# or
 
-```bash
-cargo build -p beerus-core --no-default-features --target wasm32-unknown-unknown --release
-```
+cargo run -p beerus-cli -- -c examples/conf/beerus.toml
 
-#### Test
-
-```bash
-cargo test --all
+# wait for server to start
+hurl examples/rpc/starknet_provenStateRoot.hurl
 ```
 
-#### Config
-
-The project requires an Ethereum node and a Starknet node. For Ethereum nodes
-you can use Alchemy (not Infura since it does not support getProof endpoint).
-
-Ethereum execution layer RPC URL (must be an Ethereum provider that supports
-the eth_getProof endpoint).
-
-Ethereum consensus layer RPC URL (must be a consensus node that supports the
-light client beacon chain api)
-
-For Starknet node for the moment you can use Infura but soon
-[verify proof](<[#62](https://github.com/keep-starknet-strange/beerus/issues/62)>)
-will be implemented in Pathfinder nodes, and so will these nodes be working as
-well.
-
-| Env Var | TOML | Mainnet | Goerli |
-| -------------  | -------------  | ------------- | ------------- |
-| ETHEREUM_NETWORK | ethereum_network | `mainnet` | `goerli(default)` |
-| ETHEREUM_EXECUTION_RPC_URL | ethereum_consensus_rpc | <https://eth-mainnet.g.alchemy.com/v2/XXXXX> | <https://eth-goerli.g.alchemy.com/v2/XXXXX> |
-| ETHEREUM_CONSENSUS_RPC_URL | ethereum_execution_rpc | <https://www.lightclientdata.org> | <http://testing.prater.beacon-api.nimbus.team> |
-| STARKNET_RPC_URL  | starknet_rpc | <https://starknet-mainnet.infura.io/v3/XXXXX> | <https://starknet-goerli.infura.io/v3/XXXXX> |
-
-To speed up the launch of the Ethereum client, it is recommended to set a more recent checkpoint. You can find one, for example, at this link: https://sync.invis.tools/.
-
-| Env Var | TOML | Mainnet |
-|---|---|----------------------------------------------------------|
-| ETHEREUM_CHECKPOINT | ethereum_checkpoint | 0x419347336a423e0ad7ef3a1e8c0ca95f8b4f525122eea0178a11f1527ba38c0f |
-
-##### Config File
-
-Beerus is configurable via a config toml. If you have set the env var
-`BEERUS_CONFIG` = `path/to/config` this will override all other environment
-variables and take configuration from values defined herein.
-Also the the cli can be directed via `beerus --config <path/to/config>`
-
-[goerli.toml](./crates/beerus-core/tests/common/data/goerli.toml)
-
-[mainnet.toml](./crates/beerus-core/tests/common/data/mainnet.toml)
-
-##### Environment Variables
-
-Beerus is configurable through environment variables.
+### Examples
 
-```bash
-cp examples/.env.example .env
-source .env
-```
-
-#### Examples
 ```bash
 cargo run -p beerus-core --example basic
 ```
```

Comment:
todo or del this line `BEERUS_CONFIG=path/to/config.toml cargo run -p beerus-core --example basic`?

Reply:
Would you want to add to this PR and get the examples running w/ the new layout?

Reply:
Sure

Reply:
While doing examples I started to think that maybe users shoud be abstracted from beerus core like rpc? 
I've created dummy PR to show what I mean. [Link](https://github.com/keep-starknet-strange/beerus/pull/547)